storage/sealer/mock/mock.go

```diff
@@ -732,7 +731,8 @@ func (m mockVerifProver) VerifyWindowPoSt(ctx context.Context, info prooftypes.W
 }
 
 func (m mockVerifProver) GenerateDataCommitment(pt abi.RegisteredSealProof, pieces []abi.PieceInfo) (cid.Cid, error) {
-	return ffiwrapper.GenerateUnsealedCID(pt, pieces)
+	pcid, _, err := commp.PieceAggregateCommP(pt, pieces)
```

Comment:
note for other reviewer here: the `GenerateUnsealedCID` originally referenced here was the same as the nonffi version which was renamed to `PieceAggregateCommP` for v2; it's different to what's called `GenerateUnsealedCID` here in that it doesn't pad out a sector for you, see note in chain/proofs/proofs.go above.